### PR TITLE
api diff for rc1 targeting packs

### PR DESCRIPTION
**Pull request by Anipik**
cc @adamsitnik @carlossanlop @Jozkee @eiriktsarpalis @layomia @buyaa-n @joperezr @krwq @kouvel @pgovind @tannergooding @steveharter @tarekgh @dotnet/ncl

@stephentoub @ericstj @danmoseley

cc @pranavkm @Tratcher @Pilchie @davidfowl @dougbu @richlander

cc @@dreddy-work  @ryalanms @merriemcgaw @AraHaan

### CODE REVIEW

File: release-notes/6.0/preview/api-diff/rc1/.Net/6.0.0-rc1_System.Text.RegularExpressions.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# System.Text.RegularExpressions
+
+``` diff
+ namespace System.Text.RegularExpressions {
+     public class Capture {
++        public ReadOnlySpan<char> ValueSpan { get; }
```
Comment (danmoseley) [inline release-notes/6.0/preview/api-diff/rc1/.Net/6.0.0-rc1_System.Text.RegularExpressions.md:6]: regex is good.

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Builder.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Microsoft.AspNetCore.Builder
+
+``` diff
+ namespace Microsoft.AspNetCore.Builder {
```
Comment (davidfowl) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Builder.md:4]: @captainsafia @halter73 @BrennanConroy

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Routing.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Microsoft.AspNetCore.Routing
+
+``` diff
+ namespace Microsoft.AspNetCore.Routing {
++    public sealed class EndpointGroupNameAttribute : Attribute, IEndpointGroupNameMetadata {
```
Comment (davidfowl) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Routing.md:5]: @captainsafia

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Core.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Microsoft.AspNetCore.Server.Kestrel.Core
+
+``` diff
+ namespace Microsoft.AspNetCore.Server.Kestrel.Core {
```
Comment (davidfowl) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Core.md:4]: @Tratcher

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Microsoft.AspNetCore.Server.Kestrel.Transport.Quic
+
+``` diff
+ namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Quic {
+     public class QuicTransportOptions {
+-        public string Alpn { get; set; }
+
++        public int Backlog { get; set; }
```
Comment (davidfowl) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.md:8]: @JamesNK  @Tratcher

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets
+
+``` diff
+ namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets {
++    public sealed class SocketConnectionContextFactory : IDisposable {
```
Comment (davidfowl) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.md:5]: @halter73

**Comment by tarekgh** [comment]
System.Diagnostics.Metrics.md LGTM. 

Also System.Diagnostics.md (TagList type) LGTM too.

**Review by stephentoub — APPROVED** [review]

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Core.md (hunk 4 above)
Comment (Tratcher) [reply release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Core.md:4]: :shipit:

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.md (hunk 5 above)
Comment (Tratcher) [reply release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.md:8]: :shipit:

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Components.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
++        public static string GetUriWithQueryParameter(this NavigationManager navigationManager, string name, TimeOnly value);
++        public static string GetUriWithQueryParameters(this NavigationManager navigationManager, IReadOnlyDictionary<string, object> parameters);
++        public static string GetUriWithQueryParameters(this NavigationManager navigationManager, string uri, IReadOnlyDictionary<string, object> parameters);
++    }
++    public class PersistentComponentState {
++        public void PersistAsJson<TValue>(string key, TValue instance);
++        public PersistingComponentStateSubscription RegisterOnPersisting(Func<Task> callback);
++        public bool TryTakeFromJson<TValue>(string key, out TValue instance);
++    }
++    public readonly struct PersistingComponentStateSubscription : IDisposable {
```
Comment (pranavkm) [inline release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Components.md:82]: Microsoft.AspNetCore.Components.* - :shipit:

**Comment by Anipik** [comment]
cc @steveharter @layomia for json related changes

File: release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_Microsoft.VisualBasic.ApplicationServices.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
++        public int MinimumSplashScreenDisplayTime { get; set; }
++    }
++    public delegate void ApplyApplicationDefaultsEventHandler(object sender, ApplyApplicationDefaultsEventArgs e);
+     public class WindowsFormsApplicationBase : ConsoleApplicationBase {
++        protected HighDpiMode HighDpiMode { get; set; }
++        public event ApplyApplicationDefaultsEventHandler ApplyApplicationDefaults;
+     }
+ }
+```
+
```
Comment (merriemcgaw) [inline release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_Microsoft.VisualBasic.ApplicationServices.md:17]: @KlausLoeffelmann look good to you?

File: release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+     }
+     public static class ProfessionalColors {
++        public static Color StatusStripBorder { get; }
+     }
+     public class ProfessionalColorTable {
++        public virtual Color StatusStripBorder { get; }
+     }
+ }
+```
+
```
Comment (merriemcgaw) [inline release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: lgtm, let's have @RussKie confirm.

Comment (dreddy-work) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: @KlausLoeffelmann , I see few VB specific APIs missing here. Can you please confirm? https://github.com/dotnet/core/pull/6712

Comment (KlausLoeffelmann) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: @dreddy-work: Which do you have in mind?

File: release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_Microsoft.VisualBasic.ApplicationServices.md (hunk 8 above)
Comment (KlausLoeffelmann) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_Microsoft.VisualBasic.ApplicationServices.md:17]: For the diffs, yes. Also @KathleenDollard.

File: release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md (hunk 9 above)
Comment (dreddy-work) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: Properties in `ApplyApplicationDefaultsEventArgs ` class.

Comment (KlausLoeffelmann) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: Yes. We breaking-changed them (omitted) intentionally. They are replaced by the `Default`-styled properties we have now. Good catch, thanks for double-checking!!

Comment (dreddy-work) [reply release-notes/6.0/preview/api-diff/rc1/WindowsDesktop/6.0.0-rc1_System.Windows.Forms.md:19]: I cross checked and they are there. It was a confusion.

**Review by dreddy-work — APPROVED** [review]
LGTM

**Review by RussKie — APPROVED** [review]

File: release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Routing.md (hunk 3 above)
Comment (captainsafia) [reply release-notes/6.0/preview/api-diff/rc1/Asp.Net/6.0.0-rc1_Microsoft.AspNetCore.Routing.md:5]: Looks good!

**Pull request merged**